### PR TITLE
Adds AI to the list of protected roles

### LINKED ISSUE #19024: Traitor AI happens too often
It undermines the sense of the AI being a tool, (a powerful and dangerous tool, but a tool) when it's going completely rogue all on its own a good percentage of the time, and in a stealthier way than a malf. Seems like it should be at the same level of restriction as traitor captain and traitor HoS. Last time a similar thing got discussed was #15289. I want to see what people think of this now.

### PR DESCRIPTION
**Pull request by ShiftyRail**
IE, it has the same reduced chance of getting traitor on a traitor roll as the implanted people - Security, ~HoP~, Captain.
Closes #19024, following Damian's suggestion.

HoP has by the way no reduced chance, another myth busted

As an aside there is an ageless comment about getting AI in The list ; ageless as in, before malf actually worked.
:cl:
- tweak: AIs have now the same reduced chance of rolling autotraitor than Security does.

### CODE REVIEW

**Comment by Chazach** [comment]
but what about uh
that time when the AI killed its target and did nothing until the escape shuttle was here and it then shocked the doors? haha good times man fucking coders removing fun

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	name = "traitor"
 	config_tag = "traitor"
 	restricted_jobs = list("Cyborg","Mobile MMI")//They are part of the AI if he is traitor so are they, they use to get double chances
-	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")//AI", Currently out of the list as malf does not work for shit
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "AI")//AI", Currently out of the list as malf does not work for shit
```
Comment (Pathid) [inline code/game/gamemodes/traitor/traitor.dm]: Might want to remove that comment then.

**Comment by Accipitermaris** [comment]
>it wasn’t a protected tator role this entire time

wew

**Comment by Chazach** [comment]
>neither is HoP

what the fugg is this

**Comment by Irradiation** [comment]
This is pure sneaky c*dering at work. While shifty may be a filthy c*der, he still tries to be a valuable member of society.

File: code/game/gamemodes/traitor/traitor.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/game/gamemodes/traitor/traitor.dm]: I wanted it so we know for posterity how lazy oldcoders were but I can do that

**Comment by SonixApache** [comment]
HoP isn't implanted.
Markdown was a mistake.

**Comment by DamianX** [comment]
I'm just worried this might prevent the removal of malf in the near future

**Comment by Accipitermaris** [comment]
>SonixApache 
>HoP isn't implanted

All HoPs have been implanted ever since non-human HoPs were removed, with the introduction of implanted human HoPs being even older than that. Even legacy non-human HoP slots start implanted now.

In addition: http://ss13.moe/index.php/poll/31

(This was attempted to be implemented but was done in a gunked way & therefore doesn’t properly apply to every gamemode as I understand it)

**Comment by SonixApache** [comment]
🤔

**Comment by Diavill** [comment]
no downvotes lads, hit the fucking merge button

**Comment by DamianX** [comment]
gotta wait 3 days, thanks celt

**Comment by jknpj** [comment]
It was over in a moment and the crowd all gathered 'round
There before them lay the body of the captain on the ground
Oh, he might have went on livin' but he made one fatal slip
When he tried to match the AI with the big iron on his hip,
Big iron on his hip

**Comment by boy2mantwicethefam** [comment]
"Captain change my lawset its ok i am not going to do human harm and i am not traitor lol"

**Pull request merged**